src/ServiceControl.Monitoring/Http/Diagrams/DiagramApiModule.cs

```diff
@@ -28,7 +28,7 @@ public MonitoredEndpointsModule(IProvideBreakdownBy<EndpointInstanceId>[] metric
             {
                 CreateMetric<EndpointInstanceId, ProcessingTimeStore>("ProcessingTime", IntervalsAggregator.AggregateTimings),
                 CreateMetric<EndpointInstanceId, CriticalTimeStore>("CriticalTime", IntervalsAggregator.AggregateTimings),
-                CreateMetric<EndpointInstanceId, RetriesStore>("Retries", IntervalsAggregator.AggregateRetries),
```

Comment:
Can `IntervalsAggregator.AggregateRetries` also be 🔥 ?

Reply:
done.